### PR TITLE
Add convert_currency to alpha_vantage

### PR DESCRIPTION
**Pull request by lausser**
**Description:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#15258

## Checklist:

- [X ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.alpha_vantage.markdown [hunk 1]
```diff
@@ -26,6 +26,10 @@ sensor:
     symbols:
     - symbol: GOOGL
       name: Google
+    - symbol: RMBS
```
Comment (frenck) [inline source/_components/sensor.alpha_vantage.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by frenck** [comment]
Thanks for the PR @lausser.
I've reviewed it and left you a comment, could you please take a look? 👍

File: source/_components/sensor.alpha_vantage.markdown (hunk 1 above)
Comment (lausser) [reply source/_components/sensor.alpha_vantage.markdown]: Please have a look, i moved the part with the new attribute to the examples section

**Review by lausser — COMMENTED** [review]
The example for the extra parameter is now in it's own section.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for updating the PR @lausser!

✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by frenck** [comment]
Do not run stale stale[bot]. The parent PR is still open.

**Pull request closed without merging**